### ISSUE TITLE
mana_regeneration doesn't work

### ISSUE THREAD

**Issue by TofPlays** [elBukkit/MagicPlugin#417]
There's a certain set of wands that I made where changing the  '/wand configure mana_regeneration [value]' doesn't work. It works on other wands though. Other than that the wand functions properly.  In the wand's file there even is the mana_regeneration option to the value that I want it to be at, but in-game it still stays the other value. (I did reload the plugin and also restart the server)
Here it is: (p.s. I have the latest 7.6.8 version of magic, but this wand was made in earlier version)

```
AirWand:
  protection:
    fall: 1.0
  version: 6
  spell_inventory:
    pull: 2
    tornado: 5
    levitate: 4
    push: 1
    whip: 0
    airscooter: 3
    grab: 3
  active_spell: whip
  mana_regeneration: 7
  mana: 250.0
  effect_color: a9ee98aa7f27,a9ee98aa7f27,a9ee98aa7f27
  page: 0
  spells:
  - whip
  - push
  - pull
  - levitate
  - tornado
  - airscooter
  creator_id: f6f9f593-d84d-4b11-a12c-c6b8e62c2196
  creator: TofPlays
  effect_particle: SPELL
  effect_particle_radius: 1.0
  effect_particle_count: 10.0
  effect_particle_data: 2.0
  effect_particle_offset: 0.5
  effect_particle_interval: 10000.0
  name: Air
  hotbar: 0
  organize: false
  bound: true
  keep: true
  track: true
  undroppable: true
  indestructible: true
  mode: inventory
  drop: toggle
  left_click: cast
  right_click: none
  swap: cycle_hotbar
  quiet: 1
  effect_bubbles: true
  attribute_slot: mainhand
  icon: wood_hoe:6
  path: master
  mana_max: 250
```

**Comment by TofPlays** [comment]
oh the spells list changed to dots, thats weird, in actuality it is with these - - -

**Comment by TofPlays** [comment]
Thanks a lot that worked. Btw a little note, when I changed the master path's mana regen and max mana to 0, the wand I was trying to change those options of, got switched over to another path by itself, I think apprentice, and set 40 mana regen and 250 mana. Just thought this might be useful to note.
Anyways, I switched all of the paths mana and regen to 0 and the wands working as desired, thanks for the help
